src/isa/riscv64/instr/priv/exec.h

```diff
@@ -23,6 +23,33 @@ def_EHelper(name) { \
 }
 
 #ifdef CONFIG_DEBUG
+#ifdef CONFIG_RV_DEBUG
+  def_SYS_EHelper(ebreak)
+#endif
+
+#ifdef CONFIG_RVH 
+#define def_ld_template(name) \
```

Comment:
changing the name into def_hld_template would be better. or it will mixed with normal load.